sdk-setup/src/rpmvalidation

```diff
@@ -74,7 +74,9 @@ populate_suites() {
     local suite_inis=()
     local unique_inis=()
     SUITES=()
-    suite_inis=($(sb2 -t "$target" find "${CONFIG_DIRS[@]}" -maxdepth 1 -name "*.ini" -type f -printf "%f\n" 2>/dev/null))
+    local query=$(printf 'cd %q 2>/dev/null \
```

Comment:
Why is cd needed here? Wouldn't just simple 'find %q -maxdepth 1 -name "*.ini" -type f -print\n' work just as well?

Reply:
> Also eliminated the suppression of find's stderr - hidding the error message made debugging this harder.

:)